sql_generators/glean_usage/common.py

```diff
@@ -33,6 +33,10 @@
     "generate", "glean_usage", "bigconfig", "skip_app_metrics", fallback=[]
 )
 
+BIGCONFIG_DEPRECATED_APPS = ConfigLoader.get(
```

Comment:
I think BIGCONFIG is specifically for bigeye
```suggestion
DEPRECATED_APPS = ConfigLoader.get(
```